### PR TITLE
fix: use send message api.

### PR DESCRIPTION
**Pull request by jiyoung-an**
## Summary
- use send message api.

### CODE REVIEW

File: src/ai/message.ts [hunk 1]
```diff
@@ -188,6 +188,23 @@ export class Messages extends FactoryBase {
     return messages;
   }
 
+  private async sendMessage(
```
Comment (platfowner) [inline src/ai/message.ts:191]: if it replaces createMessageAndRun(), createMessage(), createRun(), and waitForRun(), how about removing them?

**Review by platfowner — APPROVED** [review]
LGTM with some comments.
Thanks!

File: src/ai/message.ts (hunk 1 above)
Comment (jiyoung-an) [reply src/ai/message.ts:191]: done! thanks!

**Comment by jiyoung-an** [comment]
Thank you for review @platfowner! now merging...

**Pull request merged**